docs/manual/source/conf.py

```diff
@@ -27,7 +27,7 @@
 author = 'Zoltán Vörös'
 
 # The full version, including alpha/beta/rc tags
-release = '4.4.0'
+release = '4.4.1'
```

Comment:
Do you use semantic versioning? If so, a move like this would call for a major version bump.

Reply:
Definitely true. Thanks for catching this!